README.md

```diff
@@ -53,24 +54,56 @@ locals {
     ]
   ])
   
-  # This sample represents the nodes that will be used for insecure setup
+  # This sample represents the nodes that will be used for unsecured setup.
   list = flatten([
     for instance in module.ec2_instances: [
        "http://${instance.public_ip}:8080"
     ]
   ])
   
-  # This sample represents the nodes that will be used for secure setup
+  # This samples represents the nodes that will be used for secure setup.
   list = [for tag in local.nodes : "https://${tag}.omermichleviz.development.run"]
+    
 }
 ```
+#### This sample represents explicit assigning approach.
```

Comment:
`RavenDB server Terraform resource parameters`

Also this is more generic and should be first in the readme.

Reply:
I agree